src/en/space-station-14/departments/science/proposals/signals-analysis.md

```diff
@@ -0,0 +1,26 @@
+2. **Signal tuning.** Once scientists have decided which signal to study, and have optionally converged multiple radio telescopes onto its origin, they can further boost the strength of the signal by tuning the telescopes to the correct frequency. This is a simple mini-game in which players turn a pair of knobs back and forth, attempting to dial in on the setting that will maximize the signal.
+3. **Signal recording.** After tuning is finished, scientists can record the signal. The speed at which the signal is recorded is proportional to its strength, with stronger signals being recorded faster than weaker ones. Once the recording is finished, the radio telescope control console will eject a disk with a copy of the signal recording on it. Scientists can then either set the recording aside and begin hunting for another signal or move to signal processing.
+4. **Signal processing.** Signal processing is conducted using the deep space spectral analyzer console. First, a disk with a signal recording from the radio telescope control console is inserted; a waveform representing the signal will appear on the UI along with an estimate of the signal quality. The initial quality of the signal will be proportional to the strength of the original signal, and it represents how much noise is present. Players can use the console's filtering and noise removal tools to clean up the signal and improve its quality. Some care needs to be taken, as aggressive noise removal may actually degrade the signal quality rather than improve it. When the player is satisfied with the signal quality, they can initiate signal processing. After a moderate span of time has elapsed, the console will eject a research point disk. The point value of this disk will be dependent on the signal source and the final quality of the signal; it can be later redeemed by depositing the disk into a tech disk console, located in the science department. Depending on the source of the signal, certain events may trigger when signal processing ends. If the signal was from the local star, a station announcement might occur forewarning of an approaching ion storm, for example. 
```

Comment:
I think this processing minigame is a bit overly vague as something to base this whole step around. Simply saying that players use filtering and removal tools to improve quality isn't really much to go off of, and i feel like you are kinda just envisioning another game's gameplay in mind and not actually explaining what it fully entails.